### DISCUSSION TITLE
Import contacts failed

### DISCUSSION THREAD

**Discussion by jekv2** [FossifyOrg/General-Discussion#656] [General]
I dunno how to fill out a bug report but here is the screenshot with the error.
![Screenshot_20240117-142100_1](https://github.com/FossifyOrg/Contacts/assets/7735249/980f3d21-06ee-4c97-990d-0a7742effdc2)

**Comment by Aga-C** [comment]
If you want to fill the bug report, go to [Issues](https://github.com/FossifyOrg/Contacts/issues), click `New Issue` and there click `Get Started` next to `Bug Report`.

**Comment by naveensingh** [comment]
@jekv2 It should be fixed in version 1.0.1

https://github.com/FossifyOrg/Contacts/releases/tag/1.0.1
